src/freedv_api.c

```diff
@@ -1593,18 +1591,9 @@ static int freedv_comprx_700(struct freedv *f, COMP demod_in_8kHz[], int *valid)
     // quisk_cfInterpDecim() modifies input data so lets make a copy just in case there
     // is no sync and we need to echo inout to output
 
-    COMP demod_in[freedv_nin(f)];
-    for(i=0; i<freedv_nin(f); i++)
-        demod_in[i] = demod_in_8kHz[i];
-
-    i = quisk_cfInterpDecim((complex float *)demod_in, freedv_nin(f), f->ptFilter8000to7500, 15, 16);
-    //if (i != f->nin)
-    //    printf("freedv_comprx decimation: input %d output %d\n", freedv_nin(f), i);
-
-    for(i=0; i<f->nin; i++)
-        demod_in[i] = fcmult(1.0/FDMDV_SCALE, demod_in[i]);
+    i = quisk_cfInterpDecim((complex float *)demod_in_8kHz, freedv_nin(f), f->ptFilter8000to7500, 15, 16);
     
```

Comment:
@srsampson This code affects FreeDV 700C - perhaps try a manual test or add a Ctest to check 700C is getting a similar BER to before?